shared/js/notification_helper.js

```diff
@@ -30,26 +30,29 @@
     // an object -> {raw: string}
     send: function nh_send(titleL10n, options) {
       return new Promise(function(resolve, reject) {
-        navigator.mozL10n.once(function() {
-          var title = getL10n(titleL10n);
+        navigator.mozL10n.once(() => {
```

Comment:
You shouldn't need `once()` here since `formatValue` already waits for the `_ready` promise to resolve. See https://github.com/mozilla-b2g/gaia/blob/722028715a56a03f327e2e70f2c32dcb6d819d4c/shared/js/l10n.js#L1727 and https://github.com/mozilla-b2g/gaia/blob/722028715a56a03f327e2e70f2c32dcb6d819d4c/shared/js/l10n.js#L1646.